subgraph.template.yaml

```diff
@@ -57,7 +57,7 @@ templates:
       abi: DisputeManager
     mapping:
       kind: ethereum/events
-      apiVersion: 0.0.3
+      apiVersion: 0.0.5
```

Comment:
some reason to not got to 0.0.6 here also?

Reply:
Oh template was not updated, I manually change the subgraph.yaml since the scripts doesn't work with Windows. But I'll change it for sure